src/server/scripts/Commands/cs_reload.cpp

```diff
@@ -436,81 +436,78 @@ class reload_commandscript : public CommandScript
-            cInfo->DifficultyEntry[1] = fields[1].GetUInt32();
-            cInfo->DifficultyEntry[2] = fields[2].GetUInt32();
-            cInfo->KillCredit[0]      = fields[3].GetUInt32();
-            cInfo->KillCredit[1]      = fields[4].GetUInt32();
-            cInfo->Modelid1           = fields[5].GetUInt32();
-            cInfo->Modelid2           = fields[6].GetUInt32();
-            cInfo->Modelid3           = fields[7].GetUInt32();
-            cInfo->Modelid4           = fields[8].GetUInt32();
-            cInfo->Name               = fields[9].GetString();
-            cInfo->SubName            = fields[10].GetString();
-            cInfo->IconName           = fields[11].GetString();
-            cInfo->GossipMenuId       = fields[12].GetUInt32();
-            cInfo->minlevel           = fields[13].GetUInt8();
-            cInfo->maxlevel           = fields[14].GetUInt8();
-            cInfo->expansion          = fields[15].GetUInt16();
-            cInfo->faction            = fields[16].GetUInt16();
-            cInfo->npcflag            = fields[17].GetUInt32();
-            cInfo->speed_walk         = fields[18].GetFloat();
-            cInfo->speed_run          = fields[19].GetFloat();
-            cInfo->scale              = fields[20].GetFloat();
-            cInfo->rank               = fields[21].GetUInt8();
-            cInfo->mindmg             = fields[22].GetFloat();
-            cInfo->maxdmg             = fields[23].GetFloat();
-            cInfo->dmgschool          = fields[24].GetUInt8();
-            cInfo->attackpower        = fields[25].GetUInt32();
-            cInfo->DamageModifier     = fields[26].GetFloat();
-            cInfo->BaseAttackTime     = fields[27].GetUInt32();
-            cInfo->RangeAttackTime    = fields[28].GetUInt32();
-            cInfo->unit_class         = fields[29].GetUInt8();
-            cInfo->unit_flags         = fields[30].GetUInt32();
-            cInfo->unit_flags2        = fields[31].GetUInt32();
-            cInfo->dynamicflags       = fields[32].GetUInt32();
-            cInfo->family             = fields[33].GetUInt8();
-            cInfo->trainer_type       = fields[34].GetUInt8();
-            cInfo->trainer_spell      = fields[35].GetUInt32();
-            cInfo->trainer_class      = fields[36].GetUInt8();
-            cInfo->trainer_race       = fields[37].GetUInt8();
-            cInfo->minrangedmg        = fields[38].GetFloat();
-            cInfo->maxrangedmg        = fields[39].GetFloat();
-            cInfo->rangedattackpower  = fields[40].GetUInt16();
-            cInfo->type               = fields[41].GetUInt8();
-            cInfo->type_flags         = fields[42].GetUInt32();
-            cInfo->lootid             = fields[43].GetUInt32();
-            cInfo->pickpocketLootId   = fields[44].GetUInt32();
-            cInfo->SkinLootId         = fields[45].GetUInt32();
+            cInfo->DifficultyEntry[0]       = fields[1].GetUInt32();
```

Comment:
Or wrong, it should start from 0
```cInfo->DifficultyEntry[0]       = fields[0].GetUInt32();```

Reply:
You are right I had an error in my thinking here, will update now